week-2/L-array-methods-2/exercise.js

```diff
@@ -15,8 +15,8 @@ var everyone = [
   "Swathi"
 ];
 
-var firstFive; // complete this statement
-var lastFive; // complete this statement
+var firstFive = everyone.slice(0, 5); // complete this statement
+var lastFive = everyone.slice(2, 7); // complete this statement
```

Comment:
Like we discussed in Mitchell, here you can just pass things that will make your code more reusable. Rather than counting the length of an array, try using everyone.length to know where to end. You can also use that to specify where to start to (for example, if you are trying to return the last five, you could do `everyone.slice(everyone.length-5, everyone.length);`